packages/vue/src/components/organisms/SfTopBar/SfTopBar.stories.js

```diff
@@ -15,7 +15,7 @@ storiesOf("Organisms|TopBar", module)
       </template>
       <template #right>
         <div style="margin-right: 27px">Location:</div>
-        <SfImage src="/assets/storybook/SfTopBar/flag.png" alt="flag"/>
+        <SfImage tabindex="0" src="/assets/storybook/SfTopBar/flag.png" alt="flag of the USA"/>
```

Comment:
we don't need tabindex here for now. We will add here language selector component